install.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+###########
+#
+# Dev workstation installation controller
+# Calls setup scripts in required order with dependency updates and ssh settings
+#
+# author: jsinger1225
+# email: jamie.singer@gmail.com
+#
+###########
+
+# updates dependencies, setup loopback ssh key, and installs ansible
+./ansible.sh
```

Comment:
In the README, it's noted that ansible.sh has to be run using sudo.  We don't want bootstrap.sh run using sudo so executing sudo install.sh isn't an option.  My recollection is that I originally tried to write ansible.sh and bootstrap.sh as one script but was blocked by this dichotomy.